mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/139 - zone UI

Closes #139 

To test:
- Play Introductory scenario
- Proceed until phase 3 where you are requested to add a physical lock to the office
- Open Zone View and add physical lock using Physical Securities policy list
- Add COMPANY to access list and remove Public from access list
- Make sure scenario proceeds as intended

Note:
- The zone map is not implemented because it doesn't gain us any functionality and so I made it a separate case
